e2e/src/usecases/DappListDisplay.js

```diff
@@ -28,14 +27,11 @@ export default DappListDisplay = () => {
     await waitFor(element(by.text(url.hostname)))
       .toBeVisible()
       .withTimeout(10 * 1000)
+    await element(by.id('WebViewScreen/CloseButton')).tap()
   })
 
   it(':ios: should correctly filter dapp list based on user agent', async () => {
     const iOSDappList = await fetchDappList('Valora/1.0.0 (iOS 15.0; iPhone)')
-    await reloadReactNative()
-    await navigateToDappList()
-    // Give a few seconds for the dapp list to load
-    await sleep(5 * 1000)
```

Comment:
this made it hard to reuse the test for Discover tab, so cleaned up to be independent of how dapps page is reached. We typically don't need a fresh load here if the above test passes.